ivy/functional/frontends/jax/numpy/logic.py

```diff
@@ -157,3 +156,8 @@ def isinf(x, /):
 def isclose(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
     a, b = promote_jax_arrays(a, b)
     return ivy.isclose(a, b, rtol=rtol, atol=atol, equal_nan=equal_nan)
+
+
+@to_ivy_arrays_and_back
+def logical_not(x, /):
```

Comment:
Can you place your function in the alphabetical order?

Reply:
yes, I put it below the logical_and function now